packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -90,11 +109,9 @@ export class GutenbergEditorPage {
 	 */
 	async dismissWelcomeTourIfPresent(): Promise< void > {
```

Comment:
Looks like we can remove the `IfPresent` bit here?

```suggestion
	async dismissWelcomeTour(): Promise< void > {
```

Reply:
Totally, a welcome change!

Reply:
Fixed in 045a0ef365865b62ae5827960ff80cc17625aad0.